.github/workflows/update-schemas.yaml

```diff
@@ -0,0 +1,62 @@
+name: Update schemas in repository
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  update-store:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.INFRA_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.INFRA_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.SMAUG_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.SMAUG_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.MORTY_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets. MORTY_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.RICK_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.RICK_SA_TOKEN }}
+        continue-on-error: true
+
+      - name: Get schemas from infra cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ secrets.BALROG_SERVER }}
+          OPENSHIFT_TOKEN: ${{ secrets.BALROG_SA_TOKEN }}
+        continue-on-error: true
```

Comment:
Turn this into a matrix strategy run, so you don't have to copy-paste so much.

Also this should help with referencing the secrets from a strategy matrix:
https://sbulav.github.io/terraform/github-actions-matrix-secrets/

Reply:
Changed this to matrix, had to also change pull request step to `gr2m/create-or-update-pull-request-action@v1`  because previous `peter-evans/create-pull-request@v4` didn't allow for new commits from different jobs (it always ended in errors) this one allows for that so for each job environment there is one commit in the PR created. Otherwise if we wouldn't have commit for each job it would mean that it would completely overwrite changes from previous job (it would force push over previous commit